crates/router/src/core/user.rs

```diff
@@ -1927,7 +1927,7 @@ pub async fn terminate_two_factor_auth(
         .change_context(UserErrors::InternalServerError)?
         .into();
 
-    if !skip_two_factor_auth {
+    if !skip_two_factor_auth || state.conf.user.force_two_factor_auth {
```

Comment:
Should it be the other way round: 

```suggestion
    if state.conf.user.force_two_factor_auth || !skip_two_factor_auth {
```

Considering the short-circuiting behavior of `||`, shouldn't `state.conf.user.force_two_factor_auth` have higher weightage over the `skip_two_factor_auth` query param?

Reply:
Yeah, config should be given more priority. I will change this.

Reply:
Business logic wise what you said it correct.

This doesn't change the API in anyway. But I will change this though.